packages/esm-patient-chart-app/src/visit-header/visit-header.component.tsx

```diff
@@ -113,6 +115,17 @@ const VisitHeader: React.FC = () => {
   const navMenuItems = useAssignedExtensions('patient-chart-dashboard-slot').map((extension) => extension.id);
   const { startVisitLabel } = useConfig();
 
+  // I thought this would tell when a form is updated
+  // const globalFormStore = getGlobalStore('ampath-form-state', {}).getState();
+
+  // const hasFormChanges = Object.values(globalFormStore).includes('ready');
+  // console.log(hasFormChanges, 'gloable-----', globalFormStore)
+
+  const store = getWorkspaceStore();
+  const state = store.getState();
+  const hasOpenForm = state.openWorkspaces?.[0]?.type === 'form';
+  const workSpaceProps = state.openWorkspaces?.[0];
```

Comment:
It's not clear to me why things are ordered this way? Also, isn't it possible to have multiple "open" workspaces? Shouldn't this code work with that situation?

Reply:
I.e., a more appropriate implementation would be something like:

```
const formWorkspace = state.openWorkspaces?.find((workspace) => workspace.type === "form");
```